src/components/Admin/Admin.js

```diff
@@ -93,13 +93,34 @@ class Admin extends Component {
   };
 
   render() {
-    const tabStyle = {
-      width: '100%',
-      animated: false,
-      textAlign: 'left',
-      display: 'inline-block',
+    const styles = {
+      tabStyle: {
+        width: '100%',
+        animated: false,
+        textAlign: 'left',
+        display: 'inline-block',
+      },
+      skillCountStyle: {
+        display: 'flex',
+        justifyContent: 'space-between',
+        marginTop: '50px',
+      },
+      skillCountMobileViewStyle: {
+        width: '300px',
+        height: '210px',
+        marginBottom: '20px',
+        fontSize: '18px',
+        fontWeight: 'bold',
+        lineHeight: '2',
+      },
     };
-
+    const { skillStats } = this.state;
+    const { totalSkills, reviewedSkills, nonReviewedSkills } = skillStats;
+    const totalSkillsCount = skillStats && totalSkills ? totalSkills : 0;
+    const reviewedSkillsCount =
```

Comment:
No need to declare a new variable for this, directly use: `skillStats && reviewedSkills ? reviewedSkills : 0`

Reply:
@fragm3 Doing that I get eslint error while pushing the commit. "Render function has a complexity of 23".